<launch>

    <node name="rt_runner" pkg="rtcf" type="rt_runner" output="screen">
        <rosparam param="mode">"wait_for_components"</rosparam>
        <rosparam param="ros_mapping_blacklist">".*tmp.*"</rosparam>
        <rosparam param="ros_mapping_whitelist">".*mapped.*"</rosparam>
        <rosparam param="num_components_expected">1</rosparam>
        <rosparam param="frequency">1.0</rosparam>
    </node>

    <node name="parameter_test" pkg="rtcf" type="rt_launcher" args="rtcf_examples ParameterHandling" output="screen">
        <rosparam param="int_param">33</rosparam>
        <rosparam param="double_param">0.56789</rosparam>
        <rosparam param="size">2</rosparam>
        <rosparam param="another_private_param">-111222333</rosparam>
    </node>

    <rosparam param="public_param">123.456789</rosparam>

</launch>

